src/app.py

```diff
@@ -1,17 +1,23 @@
 from flask import Flask, render_template
 from dotenv import load_dotenv
+from flask_migrate import Migrate
 
 load_dotenv()
 
 from utils.config import Config
 from routes.auth_routes import auth_routes
-from db.db import database
+from routes.character_routes import character_routes
+from db.db import db
```

Comment:
It seems odd to me to have three things on different levels to have the same name. Consider re-exporting db in __init__.py.

Reply:
Yeah, good point. I'm only just learning Python and Flask so some of the imports are of the type "if it works, don't touch it." Then again, I do like odd things...